fetch-api/js/app.js

```diff
@@ -0,0 +1,70 @@
+const select = document.getElementById('breeds');
+const card = document.querySelector('.card');
+
+select.addEventListener('change', fetchBreedImage);
+card.addEventListener('click', fetchBreedImage);
+document.querySelector('form').addEventListener('submit', postInputFieldData);
+
+Promise.all([
+    fetchData('https://dog.ceo/api/breeds/image/random'),
+    fetchData('https://dog.ceo/api/breeds/list')
+])
+    .then(data => {
+        generateImage(data[0].message);
+        generateBreedOptions(data[1].message);
+    });
+
+//Functions:
```

Comment:
Dont do comments like this.
It doesnt add anything to the code - we know the functions are next since it is prefixed with the function keyword. There is not a need to explicitly call out where functions are starting.

Reply:
Removed.